.github/ISSUE_TEMPLATE.md

```diff
@@ -12,7 +12,8 @@
  - Documentation Report
 
 ##### COMPONENT NAME
-<!--- Write the short name of the module, plugin, task or feature below -->
+<!--- Write the short name of the module, plugin, task or feature below,
```

Comment:
Perhaps add components should be `single terms only`?

Reply:
Yes, I got someone today describing the feature in short, which is correct if you read the instruction.